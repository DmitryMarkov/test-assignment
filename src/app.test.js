import React from 'react'
import { render, waitForElement } from '@testing-library/react'
import App from './app'

test('renders app properly', async () => {
  const { asFragment, getByTestId } = render(<App />)
  const headingTitle = await waitForElement(() => getByTestId('heading-title'))

  expect(headingTitle).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
