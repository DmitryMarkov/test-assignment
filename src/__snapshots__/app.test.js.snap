// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app properly 1`] = `
<DocumentFragment>
  <h1
    class="css-6qc1t1"
    data-testid="heading-title"
  >
    Mallorca: 16326 Unterkünfte gefunden
  </h1>
  <div
    class="css-sk40cj"
  >
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            SA COVETA
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            SA TANCA (MANCORINA)
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            ESPAIOSA
          </h3>
          <div
            class="css-65w75"
          >
            7 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            EL PUJOL
          </h3>
          <div
            class="css-65w75"
          >
            10 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Ferienwohnung La Cabanya in Standnähe mit 1 Schlafzimmer, WLAN, Balkon, Terrasse und Gemeinschaftspo...
          </h3>
          <div
            class="css-65w75"
          >
            2 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Ruhige Lage in Strandnähe – Casa de la playa
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            CARLES
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Mit Panoramablick und privatem Pool - Finca S'Alqueria Vell
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            ES MIRADOR
          </h3>
          <div
            class="css-65w75"
          >
            4 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            BAHAMAS 1
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            CAN TIANET
          </h3>
          <div
            class="css-65w75"
          >
            10 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            PORTA DE LA MAR
          </h3>
          <div
            class="css-65w75"
          >
            5 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            BOSTON
          </h3>
          <div
            class="css-65w75"
          >
            4 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            ES DESAIGUA
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            ROSA DELS VENTS 2C
          </h3>
          <div
            class="css-65w75"
          >
            4 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            VILLA FERNANDO
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            TAMARELL
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            SES PASADORES
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            LOS GERANIOS
          </h3>
          <div
            class="css-65w75"
          >
            9 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Modernes Ferienhaus mit Garten, Terrasse, Balkon, Meerblick und WLAN; Garage vorhanden, Haustiere er...
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            PASCUCAN
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Apartment 60m²
          </h3>
          <div
            class="css-65w75"
          >
            4 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Finca 220m²
          </h3>
          <div
            class="css-65w75"
          >
            9 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Wunderschönes Ferienwohnung in unmittelbarer Nähe zum Strand mit Balkon, Terrasse, WLAN und Klimaanl...
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            BARCELO-DANUS
          </h3>
          <div
            class="css-65w75"
          >
            4 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            In den Bergen mit Meerblick - Casa Can Sans
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Mit Pool und Bergblick - Finca Ca Na Pepa
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            SON DEL RIO VELL
          </h3>
          <div
            class="css-65w75"
          >
            6 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            ONDATEGUI 47
          </h3>
          <div
            class="css-65w75"
          >
            9 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-i65ldc"
    >
      <div
        class="css-18sbqny"
      >
        <img
          class="css-9taffg"
          src="https://img.holidu.com/images/2691944d-d1ba-4a8a-8b30-4f50d1d75156/t.jpg"
        />
        <div
          class="css-1f9rp2t"
        >
          <h3
            class="css-d1eliv"
          >
            Haus am Meer mit Pool – Casa Juana Maria
          </h3>
          <div
            class="css-65w75"
          >
            8 pers.
          </div>
          <button
            class="css-1az93wp"
            disabled=""
          >
            Click me
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
